### ISSUE TITLE
Excessively large html spec file, disable embedded images

### ISSUE THREAD

**Issue by starksm64** [eclipse-ee4j/jca-api#122]
It turns out that the https://jakarta.ee/specifications/cdi/3.0/jakarta-cdi-spec-3.0.html page is too large to be served by the EF infrastructure currently. The main reason is that the html generation is embedding the image files into the single html file. This comes from the `<data-url />` element in this configuration section:

```xml
                    <execution>
                        <id>asciidoc-to-html</id>
                        <phase>generate-resources</phase>
                        <goals>
                            <goal>process-asciidoc</goal>
                        </goals>
                        <configuration>
                            <backend>xhtml</backend>
                            <attributes>
                                <doctype>book</doctype>
                                <status>${status}</status>
                                <data-uri />
                                <icons>font</icons>
                                <toc>left</toc>
                                <icons>font</icons>
                                <sectanchors>true</sectanchors>
                                <idprefix />
                                <idseparator>-</idseparator>
                                <docinfo1>true</docinfo1>
                            </attributes>
                        </configuration>
                    </execution>
```

With that enabled, the generated html file is ~59Mb:
```bash
└> ls -l target/generated-docs 
total 216032
drwxr-xr-x  205 starksm  staff      6560 Aug 25 09:27 images
-rw-r--r--    1 starksm  staff  59310423 Aug 25 09:27 jakarta-connectors-spec.html
-rw-r--r--    1 starksm  staff  50446314 Aug 25 09:27 jakarta-connectors-spec.pdf
```

With that setting removed, the generated html file is 1.1Mb:
└> ls -lh target/generated-docs
total 102536
drwxr-xr-x  205 starksm  staff   6.4K Aug 25 09:29 images
-rw-r--r--    1 starksm  staff   1.1M Aug 25 09:29 jakarta-connectors-spec.html
-rw-r--r--    1 starksm  staff    48M Aug 25 09:29 jakarta-connectors-spec.pdf

The image files are also excessively large and can be scaled down with no loss in clarity if desired to reduce the pdf size. One example is the conn-79.jpg file which is 5.3Mb and 4951 × 6514 pixels. Just converting this to a lower quality image with the same size and dpi reduces the size to 667Kb with no apparent loss of image quality.

**Comment by starksm64** [comment]
This will be obsolete if #124 is merged

**Comment by smillidge** [comment]
#124 I still go to review but should be complete this weekend

**Comment by smillidge** [comment]
Fixed by #124
